openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/DataQualityTab.tsx

```diff
@@ -173,12 +188,14 @@ const DataQualityTab: React.FC<DataQualityTabProps> = ({
                 <Tooltip
                   placement="bottomRight"
                   title={
-                    hasAccess ? t('label.edit') : NO_PERMISSION_FOR_ACTION
+                    testCaseEditPermission
+                      ? t('label.edit')
```

Comment:
We don't need button tooltip if it's not disabled since button also mentioning it's action